frontend/src/lib/AcmTimestamp.tsx

```diff
@@ -0,0 +1,46 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { css } from '@emotion/css'
+import { SimpleTimestamp } from './SimpleTimestamp'
+import { PluginContext } from './PluginContext'
+import { useContext } from 'react'
+import { TimestampProps } from '@openshift-console/dynamic-plugin-sdk'
+
+type AcmTimestampProps = TimestampProps & {
+  showIcon?: boolean
+  relative?: boolean
```

Comment:
The `relative` prop is only used by the fallback implementation that we use for local development. Since it doesn't affect the plugin version, I don't think we should offer it. It makes standalone behave differently from plugin version:
![image](https://github.com/user-attachments/assets/02f1bb9c-f6d7-48bf-ab28-709c59246dff)

In the plugin, the relative time would only be displayed for times within about 10 minutes of the current time, if the bug I found gets fixed. But currently, a relative time is never shown.

So I don't think it's useful to support relative times in the fallback implementation. If we did, it should behave in the same way as in the plugin, only being used for dates that at 10 minutes old.